app/javascript/packs/building_map.js

```diff
@@ -0,0 +1,164 @@
+let currentLocation;
+const YOUR_LOCATION_MAGIC_STRING = "Your location" // TODO: Change this!!!
+
+const start_input_field = document.getElementById("start_input");
```

Comment:
What I can say about this file is that if we have the benefit of having jQuery installed, we probably should use it uniformly when possible, i.e. using `$("#start_input")` instead of `document.getElementById` here.

Reply:
Somehow this doesn't seem to work here.

Reply:
Maybe you have to escape it like `$("#start\_input")`?
[Here is the documentation](https://api.jquery.com/id-selector/)